source/server/spongineer/incompatible.rst

```diff
@@ -52,6 +52,25 @@ Hammer Core
 
 - Solution: Change ``<entry key="World.ITickable.Override">true</entry>`` to ``false`` in the ``/asm/hammercore.xml`` file
 
+ICBM Classic
+~~~~~~~~~~~~
+- Versions: All
+- Problem: Missiles stop in midair and appear to lag in place.
+- Solution: Add the following mod specific configuration to the ``sponge/global.conf`` entity-activation-range-> mods section
```

Comment:
``entity-activation-range`` -> ``mods``

Reply:
It might be simpler just to say "Modify the ``sponge/global.conf`` file as below:"
and include the wrapping sections there (as ST-DDT suggests below).